roles/nvidia_gpu_exporter/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+_nvidia_gpu_exporter_go_ansible_arch: "{{ {'i386': 'i386',
+                                 'x86_64': 'x86_64',
+                                 'aarch64': 'arm64',
+                                 'armv7l': 'armv7',
+                                 'armv6l': 'armv6'}.get(ansible_architecture, ansible_architecture) }}"
+_nvidia_gpu_exporter_repo: "utkuozdemir/nvidia_gpu_exporter"
+_github_api_headers: "{{ {'GITHUB_TOKEN': lookup('ansible.builtin.env', 'GITHUB_TOKEN')} if (lookup('ansible.builtin.env', 'GITHUB_TOKEN')) else {} }}"
```

Comment:
```suggestion
_nvidia_gpu_exporter_github_api_headers: "{{ {'GITHUB_TOKEN': lookup('ansible.builtin.env', 'GITHUB_TOKEN')} if (lookup('ansible.builtin.env', 'GITHUB_TOKEN')) else {} }}"
```

Reply:
`_github_api_headers` is used for all other roles. Do we need to use another variable name for this role?

Reply:
It's something that needs to be changed in the other roles. The reason for it is to comply with the `var-naming[no-role-prefix]` ansible-lint rule which I'm working on in #385